packages/enrichPrescriptions/src/statusUpdates.ts

```diff
@@ -109,20 +120,37 @@ export function applyStatusUpdates(searchsetBundle: Bundle, statusUpdates: Statu
 
     logger.info(`Applying updates for prescription ${prescriptionID}.`)
 
-    const prescriptionUpdate = statusUpdates.prescriptions.filter(p => p.prescriptionID === prescriptionID)[0]
+    const prescriptionUpdate = statusUpdates.prescriptions.filter((p) => p.prescriptionID === prescriptionID)[0]
     if (!prescriptionUpdate || !prescriptionUpdate.onboarded) {
       logger.info(`Supplier of prescription ${prescriptionID} not onboarded. Applying default updates.`)
-      medicationRequests?.forEach(medicationRequest =>
+      medicationRequests?.forEach((medicationRequest) => {
+        if (delayWithPharmacyStatus(medicationRequest)) {
+          logger.info(
+            `Delaying 'With Pharmacy but Tracking not Supported' status ` +
+              `for prescription ${prescriptionID} line item id ${medicationRequest.id}`
```

Comment:
The line item id should be the identifier field not the id field.

Reply:
909b9e04f41c4cb3bfd0d7c9cee34e4f3d5044ec